lab-cortney/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This is a good start, there are two things you can improve on.  Since you're using mocha you can specify your "scripts" > "test" to be "mocha" instead of "echo \"Error: no test specified\" && exit 1" doing so allows users to run the command "npm test" when in the root directory and having that automatically run the command "mocha".  The second field which can be improved is "author" which can just be your name or email though I think this format is preferable:
youname <youremail@domain.whatever>

Also always keep the license in mind, ISC is fine, just make sure it is the one you want to use (just mentioning because it's the default).